### PR TITLE
Port TUFTestFixtureUnsupportedDelegation to FixtureBuilder

### PR DESCRIPTION
**Pull request by phenaproxima**

### CODE REVIEW

File: generate_fixtures.py [hunk 1]
```diff
@@ -185,20 +186,6 @@ def __init__(self):
             ['root', 'snapshot', 'targets', 'timestamp'])
         self.write_and_publish_repository()
 
-class TUFTestFixtureUnsupportedDelegation(TUFTestFixtureSimple):
-    # Sets up a repo using `path_hash_prefixes` which is currently not supported.
```
Comment (tedbow) [inline generate_fixtures.py:189]: We don't have this comment in the new `__init__.py` file. This is the important part about why are creating this fixture.

**Review by tedbow — CHANGES_REQUESTED** [review]
Just need to copy over the 1 comment otherwise looks good.

File: generate_fixtures.py (hunk 1 above)
Comment (phenaproxima) [reply generate_fixtures.py:189]: Fixed!

**Review by tedbow — APPROVED** [review]
The comment was added so looks good!

The fixture generation test passed so I assume everything else will.

approved if tests are Gree

**Pull request merged**